releasenotes/notes/prepare-0.45-50118e5e6d110302.yaml

```diff
@@ -0,0 +1,20 @@
+---
+prelude: |
+  The Qiskit Terra 0.45.0 release highlights are:
+
+  * We have introduced two new classes: :class:`.SingletonInstruction` (subclass of :class:`~.circuit.Instruction`)
+    and :class:`~.SingletonGate` (subclass of :class:`~.Gate`). These classes use a single instance for all objects
+    of that type, reducing the memory and construction overhead of using multiple gates in a circuit. The tradeoff,
+    however, is that they have a global shared state, which is incompatible with gates that have unique and/or
+    mutable states stored in an instance. See feature notes for more details.
```

Comment:
I think the bigger highlight here is not the introduction of the classes, but the fact the standard gates which don't take a parameter are now singletons.